pkg/gardenlet/apis/config/types.go

```diff
@@ -16,6 +16,7 @@ import (
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
 // GardenletConfiguration defines the configuration for the Gardenlet.
+// Note: Most fields need to be pointers for merging with the parent gardenlet configuration
```

Comment:
```suggestion
// GardenletConfiguration defines the configuration for the Gardenlet.
//
// Note: Most fields need to be optional (pointers) to allow ManagedSeed's gardenlet configuration
// to be merged with the parent gardenlet configuration.
// For more information, see the ManagedSeed's '.spec.gardenlet.mergeWithParent' field.
```

Reply:
Can you also add the same suggestion to the external/versioned gardenlet config? Ref https://github.com/gardener/gardener/blob/73b3d774efa71c400a6fe1d1f83b3cf534363fdc/pkg/gardenlet/apis/config/v1alpha1/types.go#L20-L21